### PR TITLE
Web Plugin: Add VueJS Language Support

### LINKED ISSUE #8202: FEATURE REQUEST: vue / vuejs language server/highlighting/coloring
### Description
add vuejs .vue files syntax highlighting and autocompletion for loaded frameworks (like onsenui <ons-* tags)

 

### Reproduction Steps

**OS and version:**    

**Diagnostics:**

### PR DESCRIPTION
**Pull request by demsking**
### What does this PR do?
Add support of the VueJS language to the Web Plugin.

### What issues does this PR fix or reference?
This fixes #8202 feature request

#### Changelog
- Add VueJS Agent
- Add Vue Support to the Web Plugin

#### Release Notes
The new version of the Web plugin adds support of the VueJS language. This includes Intellisense and highlighting.

#### Docs PR
Changes don't need doc.

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ci-test

File: .gitignore [hunk 1]
```diff
@@ -60,6 +60,7 @@ Icon?
 Thumbs.db
 */overlays
 *~
+.directory
```
Comment (skabashnyuk) [inline .gitignore:63]: why do you need this?

**Comment by skabashnyuk** [comment]
CC @tsmaeder

**Review by tolusha — APPROVED** [review]

**Comment by skabashnyuk** [comment]
@demsking you need to fix it

```
10:07:32 [INFO] --- license-maven-plugin:3.0:check (check-headers) @ ls-vue-agent ---
10:07:32 [INFO] Checking licenses...
10:07:32 [WARNING] Missing header in: /home/codenvy/workspace/che-pullrequests-test-ocp/agents/ls-vue/src/main/resources/installers/1.0.0/org.eclipse.che.ls.vue.script.sh
10:07:33 [JENKINS] Archiving disabled
```

**Comment by demsking** [comment]
@skabashnyuk The [org.eclipse.che.ls.vue.script.sh](https://github.com/eclipse/che/blob/8f4483ecbedbbcf032bc8ad1a0cac86ca3f0a6ef/agents/ls-vue/src/main/resources/installers/1.0.0/org.eclipse.che.ls.vue.script.sh#L3-L13) already have a license header:
```sh
#!/bin/bash

#
# Copyright (c) 2012-2018 Red Hat, Inc.
# This program and the accompanying materials are made
# available under the terms of the Eclipse Public License 2.0
# which is available at https://www.eclipse.org/legal/epl-2.0/
#
# SPDX-License-Identifier: EPL-2.0
#
# Contributors:
#   Sébastiend Demanou - initial API and implementation
#

is_current_user_root() {
    test "$(id -u)" = 0
}
...
```

**Comment by skabashnyuk** [comment]
@demsking then you need to add  ```org.eclipse.che.ls.vue.script.sh```  to exclude of maven license plugin. Without that, we can't merge your changes because maven can't build this project.

**Comment by skabashnyuk** [comment]
ci-test

**Comment by skabashnyuk** [comment]
@eclipse/eclipse-che-qa can you take a look?

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1457/) doesn't show any regression against this Pull Request.

**Comment by skabashnyuk** [comment]
ci-build

**Comment by dneary** [comment]
Thank you for this awesome patch @demsking! I have added a note to the next release's release notes about this. Very cool! https://github.com/eclipse/che-docs/pull/669

**Pull request merged**